sessiontxn/interface.go

```diff
@@ -147,6 +151,10 @@ type TxnManager interface {
 	GetStmtForUpdateTS() (uint64, error)
 	// GetContextProvider returns the current TxnContextProvider
 	GetContextProvider() TxnContextProvider
+	// GetReadSnapshot get snapshot with read ts
+	GetReadSnapshot() (kv.Snapshot, error)
```

Comment:
is `GetSnapshotWithStmtReadTS` , `GetSnapshotWithStmtForUpdateTS` more clear ?

Reply:
Yes